app/facades/foods_facade.rb

```diff
@@ -0,0 +1,16 @@
+class FoodsFacade
+
+  class << self
+
+    def get_foods(food)
+      json = FoodsService.find_food(food)[:foods]
+
+      json[0..9].map {|food| Food.new(food)}
+    end
+
+    def count_foods(food)
+      json = FoodsService.find_food(food)[:totalHits]
```

Comment:
Can you figure out how to grab total hits with just one API call (i.e. get this data point and the array of foods in one method)

hint: you can return more than one object from Ruby methods. 
Next week, we'll also learn about caching which would make your current setup awesome because you could keep the separate of getting foods and counting foods to separate methods, but you would retrieve totalHits from the cache rather than having to make another API call